document/js-api/index.bs

```diff
@@ -225,6 +229,7 @@ Each [=agent=] is associated with the following three [=ordered map=]s:
     * The <dfn>Memory object cache</dfn>, mapping [=memory address=]es to {{Memory}} objects.
     * The <dfn>Table object cache</dfn>, mapping [=table address=]es to {{Table}} objects.
     * The <dfn>Exported Function cache</dfn>, mapping [=function address=]es to [=Exported Function=] objects.
+    * The <dfn>Global object cache</dfn>, mapping [=global address=]es to {{Global}} objects.
```

Comment:
Presumably preceding text should now say "the following _four_ ordered maps".

Reply:
Done.

Reply:
Also FYI: https://github.com/WebAssembly/spec/pull/657